src/main/java/com/starkinfra/IndividualIdentity.java

```diff
@@ -0,0 +1,691 @@
+package com.starkinfra;
+
+import com.starkinfra.utils.Rest;
+import com.starkinfra.utils.Resource;
+import com.starkinfra.utils.Generator;
+import com.starkinfra.utils.SubResource;
+import com.starkinfra.error.ErrorElement;
+
+import java.util.Map;
+import java.util.List;
+import java.util.HashMap;
+import java.util.ArrayList;
+
+
+public final class IndividualIdentity extends Resource {
+    /**
+     * IndividualIdentity object
+     * <p>
+     * IndividualIdentities are used to aggregate an object list of IndividualDocuments
```

Comment:
IndividualIdentities are used to aggregate a list of IndividualDocument objects for natural person documents check.

Não fica melhor?